mysql-test/t/status.test

```diff
@@ -319,6 +319,10 @@ let $wait_condition =
 #
 # Bug#41131 "Questions" fails to increment - ignores statements instead stored procs
 #
+SET GLOBAL log_output = 'TABLE';
```

Comment:
That is already done by the surrounding testcase, so I only restore to what the testcase sets it to previously. But what you suggest is more robust if the surrounding testcase changes